webview-ui/src/components/settings/ExperimentalFeature.tsx

```diff
@@ -26,14 +18,9 @@ export const ExperimentalFeature = ({
 	return (
 		<div>
 			<div className="flex items-center gap-2">
-				{/* kilocode_change start */}
-				<VSCodeCheckbox
-					checked={enabled}
-					disabled={!isAvailable}
```

Comment:
This does mean the experiment is enablable when the kilocode provider isn't used. Is the UI clear that that won't work (or did you change that?)